README.md

```diff
@@ -325,6 +325,7 @@ A complete API definition is in the works. Meanwhile, you can learn how to you u
 - [miscellaneous operations](https://github.com/ipfs/interface-ipfs-core/tree/master/SPEC/MISCELLANEOUS.md)
   - [`ipfs.id([callback])`](https://github.com/ipfs/interface-ipfs-core/tree/master/SPEC/MISCELLANEOUS.md#id)
   - [`ipfs.version([callback])`](https://github.com/ipfs/interface-ipfs-core/tree/master/SPEC/MISCELLANEOUS.md#version)
+  - [`ipfs.shutdown([callback])`](https://github.com/ipfs/interface-ipfs-core/tree/master/SPEC/MISCELLANEOUS.md#shutdown)
```

Comment:
I'm thinking more about this. `shutdown` is a c&p of `stop`. Can we use just stop instead?

Reply:
https://github.com/ipfs/interface-ipfs-core/pull/215

Reply:
Yes we could change it to `stop`.  It's not a c&p, it's even simpler: `this.shutdown = this.stop`.

However, the `/api/v0/shutdown` end-point is already defined by `go`, so we must support it.  

Also what do we do about the CLI; `jsipfs shutdown` or `jsipfs stop` or both?

Reply:
@richardschneider let's do the `this.shutdown = this.stop` for now and keep the CLI as just shutdown.

Note, the HTTP API doesn't have to match entirely to the Core API which doesn't have to match the CLI API. It should enable the users to do the same operations, but we do have to have into account that they are used in different contexts. We will need to revisit all of the APIs later.

Reply:
@diasdavid I'm very confused. Is this PR now okay?  What about https://github.com/ipfs/js-ipfs-api/pull/685?

Reply:
Almost.

The spec should be `ipfs.stop` as it is on https://github.com/ipfs/interface-ipfs-core/pull/215/files and we can have an alias for `ipfs.shutdown` in both js-ipfs and js-ipfs-api just for the sake of convenience.

Similar to how we do alias on https://github.com/ipfs/js-ipfs-api/#files